qdax/tasks/brax_envs.py

```diff
@@ -0,0 +1,341 @@
+import functools
+from functools import partial
+from typing import Callable, Optional, Tuple
+
+import brax.envs
+import flax.linen as nn
+import jax
+import jax.numpy as jnp
+from typing_extensions import TypeAlias
+
+import qdax.environments
+from qdax import environments
+from qdax.core.neuroevolution.buffers.buffer import QDTransition
+from qdax.core.neuroevolution.mdp_utils import generate_unroll
+from qdax.core.neuroevolution.networks.networks import MLP
+from qdax.types import (
+    Descriptor,
+    EnvState,
+    ExtraScores,
+    Fitness,
+    Genotype,
+    Params,
+    RNGKey,
+)
+
+PlayStepFnType: TypeAlias = Callable[
```

Comment:
Don't you think we should keep this explicit in the functions typings?

Reply:
Agreed. Changes were applied